websocket_service.go

```diff
@@ -24,7 +26,7 @@ type WsPartialDepthHandler func(event *WsPartialDepthEvent)
 // WsPartialDepthServe serve websocket partial depth handler with a symbol
 func WsPartialDepthServe(symbol string, levels string, handler WsPartialDepthHandler, errHandler ErrHandler) (doneC, stopC chan struct{}, err error) {
 	endpoint := fmt.Sprintf("%s/%s@depth%s", baseURL, strings.ToLower(symbol), levels)
-	cfg := newWsConfig(endpoint)
+	cfg := newWsConfig(endpoint, timeout)
```

Comment:
if timeout cannot be changed, it's not a good idea to pass it as a required argument to `newWsConfig`, use optional argument instead?